examples/react-router/app/components/recipe_adder.jsx

```diff
@@ -0,0 +1,66 @@
+var React = require("react"),
+    Router = require("react-router"),
+    RouteHandler = Router.RouteHandler,
+    Link = Router.Link,
+    Navigation = Router.Navigation,
+    State = Router.State,
+    Fluxxor = require("../../../../");
+
+var RecipeForm = require("../forms/recipe_form.jsx");
+
+var RecipeAdder = React.createClass({
+  mixins: [
+    Fluxxor.FluxMixin(React),
+    State,
+    Navigation
+  ],
+
+  render: function() {
+    return this.renderWithLayout(
+      <div>
+        <form onSubmit={this.onSubmit}>
+          <RecipeForm ref="form" />
+          <input type="submit" value="Save" />
+        </form>
+      </div>
+    );
+  },
+
+  renderWithLayout: function(content) {
+    return (
+      <div>
+        {content}
+        <hr />
+        <Link to="home">Home</Link>
+        {" | "}<Link to="add-recipe">Add New Recipe</Link>
+      </div>
+    );
+  },
+
+  onSubmit: function(e) {
+    e.preventDefault();
+
+    var newRecipe = this.refs.form.getValue();
+    if (newRecipe) {
+      this.getFlux().actions.recipes.add(
+        newRecipe.name,
+        newRecipe.description,
+        newRecipe.ingredients,
+        newRecipe.directions,
+        function(recipe) {
```

Comment:
Thanks for the feedback, folks. I've opted updated the example in 6ed74294d9b6c7a2b7b61ffce4dea2b2c6be946d to use a store to handle the redirection, though the techniques here are interesting and certainly valid.